### DISCUSSION TITLE
Leave a menu always open

### DISCUSSION THREAD

**Discussion by cubecleveland** [JedWatson/react-select#4866] [Q&A]
Hello,

Using the async select ? Is there any way to leave the menu always open so the options are always bein shown in a fixed height size.

**Comment by ebonow** [comment]
Sounds like you are looking for the boolean prop `menuIsOpen` 

Please note that setting this to false when closing it will also prevent it from opening again, so when you want to close the menu instead set its value back to `undefined`
